assets/js/modules/adsense/datastore/sites.test.js

```diff
@@ -0,0 +1,218 @@
+				registry
+					.dispatch( MODULES_ADSENSE )
+					.receiveGetSites( fixtures.sites, { accountID } );
+
+				const sites = registry
+					.select( MODULES_ADSENSE )
+					.getSites( accountID );
+
+				await subscribeUntil( registry, () =>
+					registry
+						.select( MODULES_ADSENSE )
+						.hasFinishedResolution( 'getSites', [ accountID ] )
+				);
+
+				expect( fetchMock ).not.toHaveFetched();
+				expect( sites ).toEqual( fixtures.sites );
+			} );
+
+			it( 'dispatches an error if the request fails', async () => {
+				const response = {
+					code: 'internal_server_error',
+					message: 'Internal server error',
+					data: { status: 500 },
+				};
+				fetchMock.getOnce(
+					/^\/google-site-kit\/v1\/modules\/adsense\/data\/sites/,
+					{ body: response, status: 500 }
+				);
+
+				const fakeAccountID = 'pub-777888999';
+				registry.select( MODULES_ADSENSE ).getSites( fakeAccountID );
+				await subscribeUntil(
+					registry,
+					() =>
+						registry
+							.select( MODULES_ADSENSE )
+							.isFetchingGetSites( fakeAccountID ) === false
+				);
+
+				expect( fetchMock ).toHaveFetchedTimes( 1 );
+
+				const sites = registry
+					.select( MODULES_ADSENSE )
+					.getSites( fakeAccountID );
+				expect( sites ).toEqual( undefined );
+				expect( console ).toHaveErrored();
+			} );
+		} );
+		describe( 'getSite', () => {
+			beforeEach( () => {
+				const accountID = fixtures.clients[ 0 ]._accountID;
+
+				// Load data into this store so there are matches for the data we're about to select,
+				// even though the selector hasn't fulfilled yet.
+				registry
+					.dispatch( MODULES_ADSENSE )
+					.receiveGetSites( fixtures.sites, { accountID } );
+			} );
+			it.each( [
+				[ 'www.example.com', fixtures.sites[ 0 ] ],
+				[ 'othersubdomain.example.com', fixtures.sites[ 0 ] ],
+				[ 'www.test-site.com', fixtures.sites[ 1 ] ],
+				[ 'some-other-tld.ie', fixtures.sites[ 2 ] ],
+			] )(
+				'finds the site in this account that matches the domain: %s',
+				async ( domain, expected ) => {
+					const accountID = fixtures.clients[ 0 ]._accountID;
+					const site = registry
+						.select( MODULES_ADSENSE )
+						.getSite( accountID, domain );
+					expect( site ).toEqual( expected );
+				}
+			);
+			it.each( [
+				[ 'non-existent-site.com' ],
+				[ 'some-other-tld.com' ],
+			] )(
+				'returns null when no site matches the given domain: %s',
+				async ( domain ) => {
+					const accountID = fixtures.clients[ 0 ]._accountID;
+					const site = registry
+						.select( MODULES_ADSENSE )
+						.getSite( accountID, domain );
+					expect( site ).toEqual( null );
+				}
+			);
+			it( 'returns null when no site matches the given domain', async () => {
+				const accountID = fixtures.clients[ 0 ]._accountID;
+
+				const site = registry
+					.select( MODULES_ADSENSE )
+					.getSite( accountID, 'www.non-existent-url.com' );
+				expect( site ).toEqual( null );
+			} );
+		} );
+		describe( 'getCurrentSite', () => {
```

Comment:
A line break here as well?
```suggestion

		describe( 'getCurrentSite', () => {
```